README.rst

```diff
@@ -70,9 +71,10 @@ Usage
 Setup and Configuration
 ~~~~~~~~~~~~~~~~~~~~~~~
 
-This plugin provides two command line options: ``--doctest-plus`` for enabling
-the advanced features mentioned above, and ``--doctest-rst`` for including
-``*.rst`` files in doctest collection.
+This plugin provides three command line options: ``--doctest-plus`` for enabling
+the advanced features mentioned above, ``--doctest-rst`` for including
+``*.rst`` files in doctest collection, and ``--doctest-ufunc`` for including
+docstrings of Numpy ufuncs.
 
 This plugin can also be enabled by default by adding ``doctest_plus = enabled``
```

Comment:
Is it useful to mention how one can ensure all tests are done in `setup.cfg`? Astropy currently has `addopts = --doctest-rst` in the `[tool:pytest]` section.